lib/internal/process/pre_execution.js

```diff
@@ -568,7 +586,8 @@ function initializePermission() {
       '--allow-worker',
     ];
     ArrayPrototypeForEach(availablePermissionFlags, (flag) => {
-      if (getOptionValue(flag)) {
+      const value = getOptionValue(flag);
+      if (ArrayIsArray(value) ? value.length : value) {
```

Comment:
Why this?

Reply:
Because when nothing is passed we get an empty array which is truthy